### PR TITLE
browser: contentcontrol: add dropdown type

### PR DESCRIPTION
**Pull request by Rash419**
Signed-off-by: Rash419 <rashesh.padia@collabora.com>
Change-Id: I178561b081f9c1cb3c7db081fcdab606c6426cc5

* Resolves: # 
* Target version: master

### CODE REVIEW

**Review by vmiklos — APPROVED** [review]
Thanks, works for me as a user. I can insert a dropdown, I can add list items, I can select a list item. I can't comment on the JS style or design, leaving that to @gokaysatir or @pedropintosilva .

**Comment by pedropintosilva** [comment]
_testing it_

**Review by pedropintosilva — APPROVED** [review]
@Rash419 I see that now we:
- Moved any insert action related to the forms from `Insert` tab to `Form` tab
- We have added `insertdropdowncontrol` and `contentcontrolproperties`

I have nothing against with these ^. Even though in the core side the tab label is different (those actions live within `Tools` tab). Here, in the context of Collabora Online, and since we do not have so many options I think it makes sense to have `Form` tab

**Review by pedropintosilva — CHANGES_REQUESTED** [review]
But within the document this doesn't seem right:
image

**Comment by pedropintosilva** [comment]
but I think we should merge this as it is and then in a separated PR tackle these styles

**Review by pedropintosilva — APPROVED** [review]

**Comment by Rash419** [comment]
> @Rash419 I see that now we:
> 
> * Moved any insert action related to the forms from `Insert` tab to `Form` tab
> * We have added `insertdropdowncontrol` and `contentcontrolproperties`
> 
> I have nothing against with these ^. Even though in the core side the tab label is different (those actions live within `Tools` tab). Here, in the context of Collabora Online, and since we do not have so many options I think it makes sense to have `Form` tab. Also moving insert actions (that are not used so often) away from the insert tab and into their own specific tab seems also reasonable

@pedropintosilva In core form controls are there in both tools and form tab

**Comment by pedropintosilva** [comment]
missing icons added in https://github.com/CollaboraOnline/online/pull/4832

File: browser/src/layer/ContentControlLayer.js [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+					'enabled': true,
+					'children': [
+						{
+							'id': 'dropdownlist',
+							'type': 'treelistbox',
+							'text': '',
+							'enabled': true,
+							'children': [
+								{
+									'id': '',
```
Comment (eszkadev) [inline browser/src/layer/ContentControlLayer.js:143]: I think we can remove these scrollbar/scrollbarbox items

Comment (Rash419) [reply browser/src/layer/ContentControlLayer.js]: thanks  : )

**Pull request merged**